csrc/lower_bank_conflict.h

```diff
@@ -43,8 +43,8 @@ namespace nvfuser {
 // way == 0 --> not applicable (for example, tensor is not a smem tensor)
 // way == 1 --> no conflict
 std::unordered_map<const Expr*, std::pair<int, int>> getBankConflictInfo(
-    kir::Kernel* kernel,
-    c10::optional<LaunchParams> launch_params = c10::nullopt,
+    const kir::Kernel* kernel,
+    LaunchParams launch_params = {},
```

Comment:
`LaunchParams` supports uninitialized value, so it does not make sense to have `c10::optional<LaunchParams>`. Before this PR, `getBankConflictInfo` returns wrong result if `LaunchParams()` is provided as `launch_params`, this PR fixes this problem.